src/M6Web/Bundle/LogBridgeBundle/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -237,19 +146,14 @@ public function getMatchFilterKey(\$route, \$method, \$status)
             }
         }
 
-        return false;
+        return '';
```

Comment:
Not a big fan of using an empty string as a "not found value" (and using falsy coercion on this, later in the code). Maybe I'm missing something, but can't we use a nullable return type ?

Reply:
Changed the code to return null instead of an empty string 👍